### ISSUE TITLE
Configuration app: Upgrade Instance does not show when it is finished

### ISSUE THREAD

**Issue by ngaruko** [medic/medic-webapp#4507]
When Upgrading Instance, the page indicates that Upgrade is in progress but does not signal when it is finished. Also, once it is finished (ie `medic webapp` has actually been upgraded, the `Upgrade Instance` link is not responding to clicks until a refresh (or navigation to another page).
Tested on `alpha.dev` .
![image](https://user-images.githubusercontent.com/6979995/39560685-48ac967e-4ef4-11e8-90e6-195b1e1c6248.png)

**Comment by SCdF** [comment]
This is a duplicate of #4505.

**Comment by SCdF** [comment]
Re-opening this as we want this to only work on this page: the rest of the admin app is online only and doesn't need to know that an upgrade is completed.

**Comment by ngaruko** [comment]
LGTM.
Tested on `alpha.dev` v3.0.0
![image](https://user-images.githubusercontent.com/6979995/40694475-d1cfb3d8-6410-11e8-96d9-f34d4feaa76a.png)
